src/api/services/incident.ts

```diff
@@ -127,12 +127,17 @@ export const getIncidentsWithParams = async (
     : `hypotheses(*,created_by(${AVATAR_INFO}),evidences(id,evidence,type,component_id))`;
 
   const responder = `responders(created_by(${AVATAR_INFO}))`;
+  const { search = "" } = params!;
+  const searchStr = search
+    ? `&or=(title.ilike.*${search}*,status.ilike.*${search}*,severity.ilike.*${search}*,type.ilike.*${search}*)`
```

Comment:
This should only search for title

Reply:
This has been resolved @moshloop, please check again.